src/application/index.tsx

```diff
@@ -182,15 +182,6 @@ export const writeData = ({
   cacheVar(cache);
 };
 
-export const handleReload = () => {
-  reloadStatus(true);
-};
-
-export const handleReloadComplete = () => {
-  reloadStatus(false);
-  client.resetStore();
```

Comment:
This behavior is still maintained, but has been moved over to the [panel](https://github.com/apollographql/apollo-client-devtools/pull/1220/files#diff-7c1d36e9e85ad36384132856e68b4e2eaf0d11d4d64e911381c8878329a5f38aR40-R42) code. We reset the queries/mutations/cache anytime we connect to the client.